client/blocks/video-editor/index.jsx

```diff
@@ -85,7 +74,7 @@ class VideoEditor extends Component {
 		}
 
 		const { media } = this.props;
-		const guid = get( media, 'videopress_guid', null );
+		const guid = media?.videopress_guid;
```

Comment:
Seems like we were using the `null` just for a boolean check, so it won't make a difference whether it's `null` (like it did before) or `undefined` (like it is after this PR)?